cmake/Modules/FindROOT.cmake

```diff
@@ -201,6 +201,10 @@ IF( ROOT_CONFIG_EXECUTABLE )
 
         ENDFOREACH()
 
+        IF(NOT(ROOT_VERSION_MAJOR LESS 6))
```

Comment:
Rephrase as `IF(NOT ROOT_VERSION_MAJOR LESS 6)` (drop extra parentheses)

Reply:
OK, fixed.